packages/babel-generator/src/buffer.js

```diff
@@ -151,6 +153,7 @@ export default class Buffer {
     let originalLine = this._sourcePosition.line;
     let originalColumn = this._sourcePosition.column;
     let originalFilename = this._sourcePosition.filename;
+    let originaIdentifierName = this._sourcePosition.identifierName;
```

Comment:
linting error on originaIdentifierName

looks like it's missing the `l` - `originaIdentifierName` -> `originalIdentifierName`